code/modules/clothing/head/beanie.dm

```diff
@@ -89,5 +89,11 @@
 	desc = "If you're going on a worldwide hike, you'll need some cold protection."
 	icon_state = "waldo_hat"
 	item_color = "waldo_hat"
+	
+/obj/item/clothing/head/beanie/rasta
+	name = "rastacap"
+	desc = "Perfect for tucking in those dredlocks."
```

Comment:
Shouldn't "dredlocks" be spelled as "dreadlocks"?

Reply:
Good catch!